src/io/exporters/fbx/FBXResult.js

```diff
@@ -0,0 +1,456 @@
+      Property: "ScalingMaxX", "bool", "",0
+      Property: "ScalingMaxY", "bool", "",0
+      Property: "ScalingMaxZ", "bool", "",0
+      Property: "GeometricTranslation", "Vector3D", "",0,0,0
+      Property: "GeometricRotation", "Vector3D", "",0,0,0
+      Property: "GeometricScaling", "Vector3D", "",1,1,1
+      Property: "LookAtProperty", "object", ""
+      Property: "UpVectorProperty", "object", ""
+      Property: "Show", "bool", "",1
+      Property: "NegativePercentShapeSupport", "bool", "",1
+      Property: "DefaultAttributeIndex", "int", "",0
+      Property: "Color", "Color", "A+",0,0,0
+      Property: "Size", "double", "",100
+      Property: "Look", "enum", "",1
+    }`;
+
+// Default materials layer
+const defaultMaterialLayer = `
+    LayerElementMaterial: 0 {
+      Version: 101
+      Name: ""
+      MappingInformationType: "AllSame"
+      ReferenceInformationType: "Direct"
+      Materials: 0
+    }`;
+
+// Default layers block
+const defaultLayerBlock = `
+    Layer: 0 {
+      Version: 100
+      LayerElement:  {
+        Type: "LayerElementNormal"
+        TypedIndex: 0
+      }
+      LayerElement:  {
+        Type: "LayerElementColor"
+        TypedIndex: 0
+      }
+      LayerElement:  {
+        Type: "LayerElementMaterial"
+        TypedIndex: 0
+      }
+    }`;
+
+/**
+ * globalSettings info in output file.
+ */
+const globalSettings = `GlobalSettings: {
+    Version: 1000
+    Properties60:  {
+      Property: "UpAxis", "int", "",1
+      Property: "UpAxisSign", "int", "",1
+      Property: "FrontAxis", "int", "",2
+      Property: "FrontAxisSign", "int", "",1
+      Property: "CoordAxis", "int", "",0
+      Property: "CoordAxisSign", "int", "",1
+      Property: "UnitScaleFactor", "double", "",1
+    }
+  }`;
+
+export default class FBXResult {
+  constructor() {
+    this._resultArray = [];
+    this._info = null;
+  }
+
+  getResult(info) {
+    this._info = info;
+    this._resultArray.push(this._writeHeader());
+    this._resultArray.push(this._writeDefinitions());
+    this._resultArray.push(this._writeObjects(info.models, info.materials));
+    this._resultArray.push(this._writeRelations());
+    this._resultArray.push(this._writeConnections()); // connections between models and materials)
+    this._info = null;
+    return this._resultArray.join('');
+  }
+
+  /**
+   * Add FBXHeader info to output file.
+   * Some fields are really confusing, but it seems that all listed fields are very informative
+   */
+  _writeHeader() {
+    const FBXHeaderVersion = 1003; // 1003 is some number which appears to present in many 6.1 ASCII files
+    const FBXVersion = 6100; // Mandatory and only supported version
+    const date = new Date();
+    const timeStampVersion = 1000;
+    const creator = `Miew FBX Exporter v${this._info.version}`; // Supposed to be an engine
+
+    return `; FBX 6.1.0 project file
+; Created by Miew FBX Exporter v${this._info.version} Copyright (c) 2015-2019 EPAM Systems, Inc.
```

Comment:
Don't you think `creator` can be used here?

Reply:
Yes, it can.